04-account-kit-manage-permissions/index.ts

```diff
@@ -0,0 +1,65 @@
+// SDK exposes all you need
+import { Chains, PublicKey, Permission } from '@[SOMETHING]kit'
+import { Account } from '@[SOMETHING]kit/account'
+import { Session } from '@[SOMETHING]kit/session'
+
+
+async function main() {
+    const session = Session.login();
+    const account: Account = await session.getAccount();
+
+    // or
+    // const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Manage account Permissions
+
+    // Get a permission
+    const permission: Permission = account.getPermission("active")
+
+    // Get its public keys
+    const publicKeys: PublicKey[] = permission.keys
+
+    // Get its accounts
+    const accounts: Account[] = permission.accounts
+
+    // Add a new permission
+    await account.addPermission({
+        session, // optional, if not defined the active session which is globally stored will be used
+        permission: Permission.from({
+            permission: 'active',
+            parent: 'owner',
+            waits: [{ wait_sec: 10, weight: 1 }],
+            keys: [PublicKey.from('PUB_K1_5rcqJ8eNgNXooYyz9PTX2a2zZBktWaWezhXYGN2hrXWqfHSWYY')]
+        }),
+    });
+
+    // Update a permission
+    await permission.update(Permission.from({
+        session, // optional, if not defined the active session which is globally stored will be used
+        permission: 'owner',
+        keys: [PublicKey.from('PUB_K1_5rcqJ8eNgNXooYyz9PTX2a2zZBktWaWezhXYGN2hrXWqfHSWYY')]
+    }));
+
+    // Remove a permission
+    await permission.remove()
```

Comment:
For example after this call completes you have a reference to an invalid permission object. I think we can detach a permission object from the account and mutate it, but once you want to update it you should be calling account.

E.g. account.removePermission('my_custom_permission') or account.updatePermission('active', newPermission)

Reply:
Makes sense!